cloudify_rest_client/deployments.py

```diff
@@ -594,7 +594,8 @@ def create(self,
                site_name=None,
                runtime_only_evaluation=False,
                labels=None,
-               display_name=None):
+               display_name=None,
+               async_create=None):
```

Comment:
Can't you default it to False and then just pass it in irrespective of whether it's not None later?

Reply:
I don't want to do this, because that would then mean the new client is unable to work with pre-6.0 servers, and there's no need to do that